MIGRATION.md

```diff
@@ -1981,6 +1983,38 @@ import { Grid, GridItem } from '@contentful/f36-components';
 </Grid>;
 ```
 
+### List
+
+The properties of the List component API have been aligned with our code [style guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).
+
+#### How to migrate your List components
+
+To migrate your `List` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
+
+`npx @contentful/f36-codemod`
+
+If you want to do it manually, you must transform your existing code as follows:
+
+```tsx
+import { List, ListItem } from '@contentful/forma-36-react-components';
+
+<List element="ul" className="className" testId="testId">
+  <ListItem>ListItem1</ListItem>
+  <ListItem>ListItem2</ListItem>
+</List>;
+```
+
+to this:
+
+```tsx
+import { List, ListItem } from '@contentful/f36-components';
+
+<List as="ul" className="className" testId="testId">
```

Comment:
hmm, maybe it should compound component at some point? 🤔

Reply:
Yes, I think so, but we can do as a minor change and deprecate the `ListItem` later on, WDYT?